go/cmd/dolt/commands/merge.go

```diff
@@ -252,6 +263,11 @@ func executeMerge(ctx context.Context, dEnv *env.DoltEnv, cm1, cm2 *doltdb.Commi
 
 		if hasConflicts {
 			cli.Println("Automatic merge failed; fix conflicts and then commit the result.")
+		} else {
+			verr = UpdateStagedWithVErr(dEnv, mergedRoot)
```

Comment:
Is it important to return a new error with a new message here, rather than just returning the result of this function? If so, document that decision with a comment

Reply:
Yes i think it's important... I left a comment: https://github.com/liquidata-inc/dolt/pull/299/files#diff-1647ea59f1bf8a37af76a621b0ea152fR267